tests/lib/rules/no-mixed-operators.js

```diff
@@ -47,7 +47,9 @@ ruleTester.run("no-mixed-operators", rule, {
         {
             code: "a * b / c",
             options: [{ allowSamePrecedence: true }]
-        }
+        },
+        "a || (b ? c : d)",
+        "(a || b) ? c : d"
```

Comment:
Would you add more valid test cases? 

- Test that this change doesn't change the default behavior. For example, `a || b ? c : d` is valid.
- Test the new option `"?:"`.

Reply:
@mysticatea I have added the tests. I was not very sure where to mention about the new option in the document. I have added examples and a note under the groups. Take a look and let me know if that works.